### PR TITLE
A smorgasboard of changes

### PR DESCRIPTION
**Pull request by aseyedia**
- Added pics, stats on dashboard
- minor impr0ovements to chart
- dropdown menu for devices
- registration page
- etc

### CODE REVIEW

File: app/__init__.py [hunk 1]
```diff
@@ -1,29 +1,30 @@
...
     db.init_app(app)
     login_manager.init_app(app)
-
-    # Initialize routes after app is created to avoid circular imports
-    init_routes(app, db, login_manager)
+    migrate.init_app(app, db)
 
     with app.app_context():
-        print("Creating tables...")
-        db.create_all()
```
Comment (Coltinho10) [inline app/__init__.py:27]: why did you remove this? It's needed to create the database on startup

Comment (aseyedia) [reply app/__init__.py:27]: Oops. Not exactly sure. I was being a little frantic last night. I'll look into and and figure it out when I get home.
